views/layouts/main.php

```diff
@@ -11,6 +11,9 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="robots" content="noindex" />
+    <meta name="googlebot" content="index" />
```

Comment:
Instead of these three lines it could be `<meta name="robots" content="none" />`